opencart 2.3.0.0 - 2.3.0.2/upload/admin/controller/extension/module/smaily_for_opencart.php

```diff
@@ -270,23 +264,23 @@ public function index() {
             $data['abandoned_status'] = $this->config->get('smaily_for_opencart_enable_abandoned');
         }
 
-        // Customer sync autoresponder.
-        if (isset($this->request->post['smaily_for_opencart_autoresponder'])) {
-            $data['autoresponder'] = json_decode(html_entity_decode($this->request->post['smaily_for_opencart_autoresponder']), true);
-        } else {
-            $data['autoresponder'] = json_decode(html_entity_decode($this->config->get('smaily_for_opencart_autoresponder')), true);
-        }
         // Customer sync additional fields.
         if (isset($this->reqest->post['smaily_for_opencart_syncronize_additional'])) {
```

Comment:
Typo. Is `reqest`, should be `request`.

Reply:
Same typo seems to be in version 3.0+ package.

Reply:
Yup missed that. Only way when that shows effect is when users saves form and causes validation error. Then selected value would be reset. Will fix that for 3.0+ package separately.

Reply:
Excellent. Thank you!